src/AutoCrudAdmin/Views/Shared/_EnityFormControlsPartial.cshtml

```diff
@@ -8,11 +10,75 @@
     .form-check-input {
         margin-left: 0.5rem;
     }
+    
+    ul.ui-autocomplete {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+        cursor: default;
+        
+        max-height:200px; 
+        max-width: 236px; 
+        
+        overflow-x:hidden;
+        overflow-y:scroll;
+    }
+    
+    .ui-helper-hidden-accessible {
+        display: none;
+    }
 </style>
 
 @foreach (var formControl in Model)
 {
-    <partial model="@formControl" name="@AutoCrudAdmin.Constants.PartialView.EntityFormControlPartial"/>
+    var autocompleteExists = Model.Count(fc => fc.DisplayName == formControl.DisplayName) > 1;
+
+    if (!autocompleteExists || formControl.FormControlType == FormControlType.Autocomplete)
+    {
+        <partial model="@formControl" name="@AutoCrudAdmin.Constants.PartialView.EntityFormControlPartial"/>
+    }
+    
+    if (formControl.FormControlType == FormControlType.Autocomplete)
+    {
```

Comment:
Why the same statement in both ifs?
`formControl.FormControlType == FormControlType.Autocomplete`

Reply:
The first check helps us display only form controls that have no autocomplete substitute or autocomplete form controls. We skip displaying the form controls that are not of type autocomplete, but have such substitute. The second check makes sure we add autocomplete functionality only to those form controls we are displaying that are of this type.